client/src/pages/Welcome/Welcome.tsx

```diff
@@ -1,17 +1,22 @@
-import Typography from '@material-ui/core/Typography';
-import Box from '@material-ui/core/Box';
 import NavBar from '../../components/Navbar/NavBar';
-import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import WelcomeSide from './WelcomeSide/WelcomeSide';
+
+import useStyles from './useStyles';
 
 export default function Welcome(): JSX.Element {
+  const classes = useStyles();
   return (
     <>
       <NavBar elevation={0} color="transparent" />
-      <Container>
-        <Box my={16}>
-          <Typography variant="h2">Find the care your dog deserves</Typography>
-        </Box>
-      </Container>
+      <Grid container>
+        <Grid item xs={12} md={6}>
+          <WelcomeSide />
+        </Grid>
+        <Grid item xs={12} md={6} className={classes.dogItem}>
+          <img className={classes.twodogs} src="/alvan-nee-T-0EW-SEbsE-unsplash.jpg" alt="Two dogs" />
```

Comment:
I would recommend renaming the image file to be meaningful and descriptive.

Optionally, you can leverage the webpack bundling in the client side to [import your images](https://create-react-app.dev/docs/adding-images-fonts-and-files/) rather than linking sources to the `public` folder (you would need to move the image into the `src` folder for this to work).

Reply:
Both done